src/app/pages/cookies/cookies-modal/cookies-modal.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { TranslateModule } from '@ngx-translate/core';
+import { instance, mock, when } from 'ts-mockito';
+
+import { COOKIE_CONSENT_OPTIONS } from 'ish-core/configurations/injection-keys';
+import { CookiesService } from 'ish-core/services/cookies/cookies.service';
+
+import { CookiesModalComponent } from './cookies-modal.component';
+
+// tslint:disable:no-intelligence-in-artifacts
+describe('Cookies Modal Component', () => {
+  let component: CookiesModalComponent;
+  let fixture: ComponentFixture<CookiesModalComponent>;
+  let element: HTMLElement;
+
+  beforeEach(async () => {
+    const cookiesServiceMock = mock(CookiesService);
+    when(cookiesServiceMock.get('cookieConsent')).thenReturn(
+      JSON.stringify({ enabledOptions: ['required', 'functional'], version: 1 })
+    );
+
+    await TestBed.configureTestingModule({
+      imports: [RouterTestingModule, TranslateModule.forRoot()],
+      declarations: [CookiesModalComponent],
+      providers: [
+        {
+          provide: COOKIE_CONSENT_OPTIONS,
+          useValue: {
+            options: [
+              {
+                id: 'required',
+                name: 'required.name',
+                description: 'required.description',
+                required: true,
+              },
+              {
+                id: 'functional',
+                name: 'functional.name',
+                description: 'functional.description',
+              },
+            ],
+          },
+        },
+        { provide: CookiesService, useValue: instance(cookiesServiceMock) },
+      ],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CookiesModalComponent);
+    component = fixture.componentInstance;
+    element = fixture.nativeElement;
+  });
+
+  it('should be created', () => {
+    expect(component).toBeTruthy();
+    expect(element).toBeTruthy();
+    expect(() => fixture.detectChanges()).not.toThrow();
+  });
```

Comment:
The easiest test extension would be `expect(element)toMatchInlineSnapshot()`, but I guess we can also skip that 😉